initializer.sh

```diff
@@ -138,7 +137,7 @@ cd ~/.radicle/seed/creature-pigeon
                     --allow $peer_one \
                     --allow $peer_two \
                     --allow $peer_three \
-                    --no-confirm 
```

Comment:
Here as well. `--allow $peer_four \` needs to be added.

Reply:
I refactored that to use an array instead. that simplifies the command.
I also put the DIDs in a file instead. that's then simpler to add/remove DIDs